templates/named.conf.options.erb

```diff
@@ -22,9 +22,9 @@ options {
 <% end -%>
 
 <% if @listen_on.size == 0 then -%>
```

Comment:
Hmm, my only problem here is this gate. You can't set the port without setting a listen_on?